### ISSUE TITLE
Map local not working with atlantis

### ISSUE THREAD

**Issue by freyzou** [ProxymanApp/atlantis#147]
Proxyman map local not working when using atlantis.

**Comment by NghiaTranUIT** [comment]
@freyzou it's expected behavior because Atlantis is built for previewing Request/Response purposes. 

If you'd like to use Map Local, please use a normal Proxy on your iPhone. You can find the Setup Guide in the Certificate menu -> Install for iOS -> Physical Device / Simulator

**Comment by freyzou** [comment]
@NghiaTranUIT For some unknown reason, some of my requests that I want to debug are not showing on Proxyman normal mode.

**Comment by NghiaTranUIT** [comment]
>  my http requests that I want to debug are not showing on Proxyman normal mode

It's odd. If you config the Proxy on your Phone correctly, it will show. You can test it by opening the Safari app -> Visit www.google.com -> Verify if you can see this traffic on Proxyman.

**Comment by freyzou** [comment]
@NghiaTranUIT N I don't think the problem is related to my Phone config, the same phone, two apps with moya, one works fine but another not.

**Comment by poos** [comment]
> @NghiaTranUIT For some unknown reasons, my http requests that I want to debug are not showing on Proxyman normal mode. I'm using Moya framework as a client. By the way, requests from my another app which also using Moya working great with Proxyman normal mode.

This case need relaunch/rebuild all.

**Comment by NghiaTranUIT** [comment]
Yes, or just update Atlantis to the latest version and double check the config:
```
Atlantis.start("Your-device-name")
```
